app/views/users/_profile.html.erb

```diff
@@ -157,14 +157,18 @@
                               aria: { describedby: "#{field.attribute_name}-description" },
                               data: { character_span_id: "#{field.attribute_name}-characters" },
                               class: "crayons-textfield") %>
-            <div id="<%= field.attribute_name %>-description" class="self-end fs-s">
-              <span class="screen-reader-only">Available characters used: </span>
-              <span id="<%= field.attribute_name %>-characters"></span>/<%= character_count_denominator(field["input_type"]) %>
+            <div style="display: grid; grid-template-columns: 85% 15%;">
+              <div>
+                <% if field.description.present? %>
+                  <p class="crayons-field__description"><%= field.description %></p>
+                <% end %>
+              </div>
+               <div id="<%= field.attribute_name %>-description" class="fs-s" style="justify-self: end;">
```

Comment:
I could not find a class that allowed me to right-justify the counter within its 15% grid column. So I added this style.